underfs/wasb/src/main/java/alluxio/underfs/wasb/WasbUnderFileSystem.java

```diff
@@ -53,8 +58,13 @@ public static Configuration createConfiguration(UnderFileSystemConfiguration con
         wasbConf.set(key, value);
       }
     }
-    wasbConf.set("fs.AbstractFileSystem.wasb.impl", "org.apache.hadoop.fs.azure.Wasb");
-    wasbConf.set("fs.wasb.impl", "org.apache.hadoop.fs.azure.NativeAzureFileSystem");
+    if (isSecure) {
+      wasbConf.set("fs.AbstractFileSystem.wasb.impl", "org.apache.hadoop.fs.azure.Wasbs");
```

Comment:
Does this not have to be "fs.AbstractFileSystem.wasbs.impl"?

Reply:
Yes this should be, that is a good catch. Strange that `wasbs` worked correctly in my testing.

Reply:
I've validated the new code works on a k8s installation. Resolving.